tensorflow/core/grappler/optimizers/remapper.cc

```diff
@@ -3174,8 +3177,18 @@ bool RuntimeFusionEnabled(const Cluster& cluster) {
         }
       }
     }
-    return CudnnUseFrontend() && CudnnUseRuntime() && num_gpus > 0 &&
-           num_gpus == num_ampere;
+    bool runtime_fusion_enabled = CudnnUseRuntimeFusion() &&
+                                  CudnnUseFrontend() && num_gpus > 0 &&
+                                  num_gpus == num_ampere;
+
+    if (CudnnUseRuntimeFusion() && !runtime_fusion_enabled) {
+      VLOG(WARNING) << "Enabling Cudnn with runtime compilation requires the "
+                    << "Cudnn frontend and Ampere GPUs or later, but we got "
+                    << "Cudnn frontend: " << CudnnUseFrontend() << " and "
+                    << "number of Ampere GPUs or later: " << num_ampere;
```

Comment:
This will be confusing on machines where there's e.g. 1 Ampere GPU and 2 GPUs overall.  Improve this error?

Reply:
Done.